### DISCUSSION TITLE
How to search music videos only?

### DISCUSSION THREAD

**Discussion by radiumrasheed** [LuanRT/YouTube.js#7] [Q&A] [answered]
on v3, you can specify `categoryId=10` for this

**Comment by LuanRT** [comment, marked as answer]
Hello, 

Currently, the library does not support searching for music videos only. However, that feature is already being worked on and will be available soon.

Update: Implemented on version 1.3.0

Regards, 
Luan.

**Reply by radiumrasheed** [reply]
thanks, if you don't mind
keep me posted on the PR
